file3.js

```diff
@@ -6,4 +6,11 @@ var def = 3;
 var ghi = 4;
 var abc = 2;
 var def = 3;
-var ghi = 4;
\ No newline at end of file
+var ghi = 4;
+
+function subtract (x,y) {
```

Comment:
This function could be optimized in a better way, can you please refactor this one.